runtime/queries/cpon/highlights.scm

```diff
@@ -26,21 +27,27 @@
 (null) @constant.builtin
 
 ; Punctuation
-
 [
   ","
   ":"
 ] @punctuation.delimiter
 
-[ "{" "}" ] @punctuation.bracket
+[
+  "{"
+  "}"
+] @punctuation.bracket
 
-[ "[" "]" ] @punctuation.bracket
+[
+  "["
+  "]"
+] @punctuation.bracket
 
-[ "<" ">" ] @punctuation.bracket
+[
+  "<"
+  ">"
+] @punctuation.bracket
 
-(("\"" @string)
- (#set! conceal ""))
+(("\"" @string)(#set! conceal ""))
```

Comment:
What happened here? :thinking: 
```suggestion
(("\"" @string)
  (#set! conceal ""))
```

Reply:
Probably backslash escape tripping things up.

Reply:
Just me didn't think there'll be a pattern of `(grouping (grouping) (node))`

Reply:
Is this something we can standardize as well? This seems like a (needlessly) unusual pattern.

Reply:
Resolved